docs/openapi/schemas.yaml

```diff
@@ -919,6 +919,154 @@ SiteTopPageList:
   type: array
   items:
     $ref: './schemas.yaml#/SiteTopPage'
+SiteExperimentVariant:
+  type: object
+  properties:
+    name:
+      description: The name of the variant
+      type: string
+    label:
+      description: The label of the variant
+      type: string
+    split:
+      description: The split allocation of the traffic to the variant
+      type: number
+    url:
+      description: The url of the variant
+      $ref: '#/URL'
+    views:
+      description: The number of views for the variant
+      type: integer
+    interactionsCount:
+      description: The interactions count for the variant
+      type: integer
+    p_value:
```

Comment:
I'm tempted to rename that to `confidence` and return `1 - p_value`

Reply:
not sure if `(1-p_value)*100` is the confidence, may be we also have to use `power` to determine the confidence. @ddragosd any feedback on this?